src/test/regress/sql/upgrade_columnar_after.sql

```diff
@@ -39,3 +39,81 @@ SELECT * FROM columnar.options WHERE regclass = 'test_options_2'::regclass;
 VACUUM VERBOSE test_options_2;
 SELECT count(*), sum(a), sum(b) FROM test_options_2;
 
+-- since we run "after schedule" twice, truncate tables to
+-- avoid getting duplicate key errors
+TRUNCATE less_common_data_types_table, columnar_with_constraints, text_data;
```

Comment:
I think this breaks the main purpose of the test that I added: Make sure that after upgrade we properly preserve all the data/indexes/constraints.

I'd be in favor of removing INSERTs in the after tests if that's the problem?

Reply:
I think the approach used so far in the other upgrade after tests is to put all stuff in the after test in a transaction that is rolled back.

Reply:
This was actually done at the bottom of this file already.